generator/generator_tool/generator_tool.cpp

```diff
@@ -228,6 +238,13 @@ int main(int argc, char ** argv)
 
     if (!FLAGS_srtm_path.empty())
       routing::BuildRoadAltitudes(datFile, FLAGS_srtm_path);
+
+    if (FLAGS_generate_routing)
```

Comment:
Yes. It's a place holder now. Routing restrictions will be generated with routing section only. So there's no need to have a special flag for routing restrictions. Routing restrictions together with other routing info will be generated when flag  generate_routing is set.
I'm adding TODO about it.